src/main/java/bio/terra/service/profile/flight/update/ProfileUpdateFlight.java

```diff
@@ -0,0 +1,35 @@
+package bio.terra.service.profile.flight.update;
+
+import bio.terra.service.iam.AuthenticatedUserRequest;
+import bio.terra.service.job.JobMapKeys;
+import bio.terra.service.profile.ProfileService;
+import bio.terra.service.resourcemanagement.google.GoogleProjectService;
+import bio.terra.stairway.Flight;
+import bio.terra.stairway.FlightMap;
+import org.springframework.context.ApplicationContext;
+
+public class ProfileUpdateFlight extends Flight {
+
+    public ProfileUpdateFlight(FlightMap inputParameters, Object applicationContext) {
+        super(inputParameters, applicationContext);
+
+        ApplicationContext appContext = (ApplicationContext) applicationContext;
+        ProfileService profileService = (ProfileService) appContext.getBean("profileService");
+        GoogleProjectService googleProjectService =
+            (GoogleProjectService) appContext.getBean("googleProjectService");
+
+        bio.terra.model.BillingProfileUpdateModel request =
+            inputParameters.get(JobMapKeys.REQUEST.getKeyName(), bio.terra.model.BillingProfileUpdateModel.class);
+
+        AuthenticatedUserRequest user = inputParameters.get(
+            JobMapKeys.AUTH_USER_INFO.getKeyName(), AuthenticatedUserRequest.class);
+
+        //update billing account id metadata
+        addStep(new UpdateProfileMetadataStep(profileService, request, user));
+        // Make sure valid account before changing in gcloud project
+        addStep(new UpdateProfileVerifyAccountStep(profileService, user));
```

Comment:
You can determine here what the flight is updating. If it is only the description, only generate the metadata step.
If it is the billing account, then generate all three steps.

You could pass that into the steps, if they need to know, rather than having them each figure it out.

Reply:
That would make a lot more sense! 

I decided to simplify my solution by requiring both a billingAccountId and description. I decided to do this for a few reasons:
- By requiring both fields, the user experience will be clear: The billing account and description will be updated according to their input. 
- There is a workaround if only one needs to be updated: The user can "retrieve" the profile, and set the billing account or description to the current value. 
- If billing account is updated, then most likely an update to the description would be useful. However, I think there may be a case for just updating the description. 
- This avoids early over-optimization. By requiring both fields, the solution is simpler, but the full functionality can be achieved. We can always customize this in the future to better fit the needs of the user.